src/github.com/travis-ci/jupiter-brain/server/server.go

```diff
@@ -172,6 +231,13 @@ func (srv *server) handleInstancesCreate(w http.ResponseWriter, req *http.Reques
 		return
 	}
 
+	instance.CreatedAt = time.Now().UTC()
+	err = srv.db.SaveInstance(instance)
```

Comment:
We should probably destroy the instance again if this returns an error, otherwise we'll leave instances lying around.